waspc/data/Generator/templates/server/src/core/auth.js

```diff
@@ -1,23 +1,10 @@
 {{={= =}=}}
-import jwt from 'jsonwebtoken'
-import SecurePassword from 'secure-password'
-import util from 'util'
 import { randomInt } from 'node:crypto'
 
 import prisma from '../dbClient.js'
 import { handleRejection } from '../utils.js'
-import HttpError from '../core/HttpError.js'
-import config from '../config.js'
-import { deserializeAndSanitizeProviderData } from '../auth/utils.js'
-
-const jwtSign = util.promisify(jwt.sign)
-const jwtVerify = util.promisify(jwt.verify)
-
-const JWT_SECRET = config.auth.jwtSecret
-
-export const signData = (data, options) => jwtSign(data, JWT_SECRET, options)
-export const sign = (id, options) => signData({ id }, options)
-export const verify = (token) => jwtVerify(token, JWT_SECRET)
+import { getSessionAndUserFromBearerToken } from '../auth/session.js'
+import { throwInvalidCredentialsError } from '../auth/utils.js'
 
 const auth = handleRejection(async (req, res, next) => {
```

Comment:
I know we previously had no docs, but it might be nice to put a comment now that says that auth attaches stuff to `req`, and which stuff.